explore-analyze/elastic-inference/inference-api.md

```diff
@@ -93,11 +93,15 @@ By default, documents are split into sentences and grouped in sections up to 250
 
 ### Chunking strategies
 
-Two strategies are available for chunking: `sentence` and `word`.
+Four strategies are available for chunking: `sentence`, `word`, `recursive` and `none`.
```

Comment:
```suggestion
Several strategies are available for chunking: 
```
Since you're listing them right after and these items are already correctly tagged, you can simplify this intro sentence